src/python/pants/engine/internals/build_files.py

```diff
@@ -104,29 +99,29 @@ async def parse_address_family(
     return AddressFamily.create(directory.path, address_maps)
 
 
-def _raise_did_you_mean(address_family: AddressFamily, name: str, source=None) -> None:
-    names = [a.address.target_name for a in address_family.addressables]
+def _did_you_mean_exception(address_family: AddressFamily, name: str) -> ResolveError:
+    names = [
+        addr.target_name or os.path.basename(addr.spec_path)
```

Comment:
This was a regression from the recent Address change. We used to always have a `.target_name`, but now it's Optional, so we could have `None` show up in the suggestions.